packages/Path-View-Core.package/SClassDiagram.class/instance/boundingBoxOfSubmorphs.st

```diff
@@ -0,0 +1,12 @@
+geometry
+boundingBoxOfSubmorphs
+	"calculate the bounding box of all morphs in the view - also notes"
+
+	| emptyRectangle |
+	(self submorphs notEmpty) ifTrue: 
```

Comment:
we should do it like this:
```smalltalk
^ (self submorphs notEmpty)
   ifTrue: [...]
   ifFalse: [ 0@0 corner: 0@0]
```
In this way it is more clear what gets returned when